projects/memapCore/src/main/java/memap/components/prototypes/Storage.java

```diff
@@ -77,7 +83,11 @@ public void handleRequest() {
 			if (storageChargeRequest!= null && storageDischargeRequest!= null) {
```

Comment:
Same topic/question as above: Shouldn't this whole part better be shifted to CSVStrorage.java ?
The abstract class Storage.java defines the needed variables (and/or mehthods) but concrete numbers (and methods) are set when instantiated by the inheriting classes *CSVStorage.java* or *ClientStorages.java*.

As long as this input from the UI is missing, storage loss should be hardcoded in CSVStorage, not in the super-class.

Also renaming *linprogResult* to *optResult* may avoid confusion here.